parallel-install/pkg/preinstaller/preinstaller.go

```diff
@@ -34,11 +36,17 @@ import (
 	"k8s.io/client-go/dynamic"
 )
 
+const (
```

Comment:
consider moving this to common file

Reply:
I will introduce some `constants.go` file.